cmd/cainjector/app/start.go

```diff
@@ -83,11 +85,11 @@ CA data from the referenced certificates, which can then be used to serve API
 servers and webhook servers.`,
 
 		// TODO: Refactor this function from this package
-		Run: func(cmd *cobra.Command, args []string) {
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Changed to `RunE` in an effort to bubble all errors back to the origin and to isolate the used of `os.Exit` to the `main.go` entry point.